azure-pipelines-templates/device-test.yml

```
@@ -0,0 +1,154 @@
+          - bash: |
+              # Define the base directory to search in
+              BASE_DIR=$(System.DefaultWorkingDirectory)/Artifacts
+              # Extract the directory path from the unitTestRunsettings parameter
+              UNIT_TEST_RUNSETTINGS_DIR=$(dirname "$BASE_DIR/${{ split(tests, ',')[0] }}")
+
+              # Find and remove all .trx files in the subdirectory
+              find "$UNIT_TEST_RUNSETTINGS_DIR"/TestResults -type f -name "*.trx" -exec rm -f {} \;
+
+              # Optional: Print a message indicating completion
+              echo "All .trx files have been removed from $unitTestRunsettings"
+            displayName: Cleaning tests results
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)